linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/rule.yml

```diff
@@ -0,0 +1,83 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure basic parameters of Audit system'
+
+description: |-
+    Perform basic configuration of Audit system.
+    Make sure that any previously defined rules are cleared, the auditing system is configured to handle sudden bursts of events, and in cases of failure, messages are configured to be directed to system log.
+
+    The following rules configure audit as described above:
+    <pre>
+    ## First rule - delete all
+    -D
+    
+    ## Increase the buffers to survive stress events.
+    ## Make this bigger for busy systems
+    -b 8192
+    
+    ## This determine how long to wait in burst of events
+    --backlog_wait_time 60000
+    
+    ## Set failure mode to syslog
+    -f 1
+    </pre>
+
+    The <tt>Audit</tt> package provides pre-configured  rules in <tt>/usr/share/audit/sample-rules</tt>. The above content can be found in <tt>/usr/share/audit/sample-rules/10-base-config.rules</tt>.
+    To deploy this configuration, it is recommended to copy it over to the <tt>/etc/audit/rules.d/</tt> directory:
+    <pre>
+    cp /usr/share/audit/sample-rules/10-base-config.rules /etc/audit/rules.d/
+    </pre>
+    
+    The file has the following SHA-256 checksum:
+    <pre>54c2cebdaadfb928f7327cc066218e38743f0ff94d02fe162a7a415e148d23a8</pre>
+    
+    Load new Audit rules into kernel by running:
+    <pre>augenrules --load</pre>
+    
+
+rationale: |-
+    Without basic configurations, audit may not perform as expected. It may not be able to correctly handle events under stressful conditions, or log events in case of failure.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 82462-3
+
+references:
+    ospp: FAU_GEN.1.1.c
+    nist: AU-2(a)
+    srg: SRG-OS-000365-GPOS-00152,SRG-OS-000475-GPOS-00220
+
+ocil_clause: 'the content or the hash of file differs or file does not exist'
+
+ocil: |-
```

Comment:
Just to be sure this is not an oversight in the discussions.
The Rule OCIL and the description contains both the file content and the checksum.